src/graph/DescribeEdgeIndexExecutor.cpp

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/DescribeEdgeIndexExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+DescribeEdgeIndexExecutor::DescribeEdgeIndexExecutor(Sentence *sentence,
+                                           ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<DescribeEdgeIndexSentence*>(sentence);
+}
+
+Status DescribeEdgeIndexExecutor::prepare() {
+    return Status::OK();
+}
+
+
+void DescribeEdgeIndexExecutor::execute() {
```

Comment:
Do we need desc for index, `show index` is enough?

Reply:
`SHOW TAG/EDGE INDEXES` will display an index list.